docs/modules/ROOT/pages/reference/database.adoc

```diff
@@ -2,17 +2,21 @@
 
 This document lists and details the database schema used by KubeArchive.
 
-== Table `kubearchive.resource`
+== Table `resource`
 
 [%header, cols="2m,2m,3"]
 |===
 |Name
 |Type
 |Description
 
+|id
+|id primary key
+|The ID of the Kubernetes resource used as primary key.
+
```

Comment:
I'm sure about this change. I thought that we were using the UUID as primary key.

Reply:
@mafh314, this should be fixed now (it's gone).